matrix-bot/src/MatrixBot/Main.hs

```diff
@@ -1,8 +1,47 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
nit: this is no longer the entrypoint. I guess you use this module to type check the whole hierarchy, so we could rename it as `Network.Matrix.Bot.Tutorial.Basic`